bench/report.txt

```diff
@@ -0,0 +1,11 @@
+Running benchmark report...
+
+Adding jobs
+....................................................................................................
+Running jobs
+....................................................................................................
+
+Adding Jobs Time:   1.5007ms
+Running Jobs Time: 11.7543ms
```

Comment:
@jcredding I'd like to see the number of jobs in this output, maybe:  `Adding {num} Jobs Time: ...` - same for the running jobs line.